tests/core/tests/test_cli_import.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import unicode_literals
+
+import os
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+
+
+class CLIImportTest(TestCase):
```

Comment:
As far as the feature goes, great work you two! 👏 

It still needs more tests though before we can approve. I was thinking that an easier way to test, instead of through the command which is a bit wonky, would be to move the logic to its own separate module.  That way we can test the functionality directly.  What do you think? 

The good news is that this feature seems to be nearly complete.  If I have some time this week Ill try to work on it here and there.  I already have a couple of changes on the branch that I'll push later on.

Reply:
Sounds good to me - thank you!